tests/qam-updinstall/update_install.pm

```diff
@@ -29,33 +29,24 @@ use qam;
 use maintenance_smelt qw(get_packagebins_in_modules get_incident_packages);
 use testapi;
 
-sub resolve_conflicts {
-    my $binary   = $_[0];
+sub has_conflict {
+    my $binary   = shift;
     my %conflict = (
         'reiserfs-kmp-default'   => 'kernel-default-base',
         'kernel-default'         => 'kernel-default-base',
         'kernel-default-extra'   => 'kernel-default-base',
-        'kernel-default-base'    => 'kernel-default',
```

Comment:
Why remove this entries ? In your VR it would try to install xen-tools-domU instead of skipping it

Reply:
I removed the entries that basically point in opposite directions to make clear which side is preferred to be installed in case of conflict.

Reply:
It was covering the conflict in both directions, try to install everything.
Now we picked side, is it the good side if there is one ? :yum:

Reply:
well, better me pick than random :)

Reply:
And of course this assumes we always release xen-tools and xen-tools-domU in one patch